pcsx2/gui/MainMenuClicks.cpp

```diff
@@ -809,75 +813,89 @@ void MainEmuFrame::Menu_Capture_Screenshot_Screenshot_Click(wxCommandEvent & eve
 #ifndef DISABLE_RECORDING
 void MainEmuFrame::Menu_Recording_New_Click(wxCommandEvent &event)
 {
-	NewRecordingFrame* NewRecordingFrame = wxGetApp().GetNewRecordingFramePtr();
-	if (NewRecordingFrame)
+	const bool initially_paused = g_RecordingControls.isRecordingPaused();
```

Comment:
As I look through again, I see instances of mixing snake_case with camelCase.  Ideally we should stick to one, and pcsx2 is trying to move in the direction of camelCase.